src/components/Inputs/SelectOptionItemContent.tsx

```diff
@@ -20,31 +20,37 @@ import { ReactNode } from 'react'
 import styled from 'styled-components'
 
 interface OptionItemContentProps {
-  ContentLeft: ReactNode
-  ContentRight?: ReactNode
+  MainContent: ReactNode
+  SecondaryContent?: ReactNode
   className?: string
 }
 
-const OptionItemContent = ({ ContentLeft, ContentRight, className }: OptionItemContentProps) => (
+const OptionItemContent = ({
+  MainContent: ContentLeft,
+  SecondaryContent: ContentRight,
+  className
+}: OptionItemContentProps) => (
   <div className={className}>
-    <OptionItemContentLeft>{ContentLeft}</OptionItemContentLeft>
-    <OptionItemContentRight>{ContentRight}</OptionItemContentRight>
+    <MainOptionContent>{ContentLeft}</MainOptionContent>
+    <SecondaryOptionContent>{ContentRight}</SecondaryOptionContent>
```

Comment:
nit: This naming sounds a bit weird to me as if it's a "main option" and "secondary option". I'd switch the order of the words to `OptionMainContent` `OptionSecondaryContent`.

Reply:
Fixed in https://github.com/alephium/desktop-wallet/pull/614